cfme/v2v/__init__.py

```diff
@@ -0,0 +1,45 @@
+from navmazing import NavigateToSibling
+from widgetastic.widget import Text
+
+from cfme.base import Server
+from cfme.base.login import BaseLoggedInPage
+from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep
+
+
+class MigrationView(BaseLoggedInPage):
+    title = Text("#explorer_title_text")
+
+    @property
+    def in_explorer(self):
+        return self.logged_in_as_current_user and (
+            self.navigation.currently_selected == ["Compute", "Migration"]
+            or self.navigation.currently_selected == ["Compute", "Migration", "Migration Plans"]
+        )
+
+    @property
+    def is_displayed(self):
+        return self.in_explorer
+
+
+@navigator.register(Server)
+class Migration(CFMENavigateStep):
+    VIEW = MigrationView
+    prerequisite = NavigateToSibling("LoggedIn")
+
+    def step(self):
+        self.view.navigation.select("Compute", "Migration")
+
+
+class InfrastructureMappingView(BaseLoggedInPage):
```

Comment:
Please check I've definitely seen such view

Reply:
Views like MigrationView and InfrastructureMappingView exists in class migrations.py which I am refactoring and splitting the code in smaller and separate class .  there are lot of tests that run on migrations.py so the plan is to refactor code and tests and once all the refactored code is in , delete the old code (migrations.py ) and related files and entry points so that jenkins tests doesn't break